man/ocitools-generate.1.md

```diff
@@ -234,6 +234,45 @@ inside of the container.
   The special *PATH* `host` removes any existing UTS namespace from
   the configuration.
 
+**--cpu-shares**=CPUSHARES
```

Comment:
Options in this man page are currently alphabetized (since #44).  That's an easier pattern to follow than grouping related options.